toolchain/check/cpp/access.h

```diff
@@ -0,0 +1,20 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_CHECK_CPP_ACCESS_H_
+#define CARBON_TOOLCHAIN_CHECK_CPP_ACCESS_H_
+
+#include "toolchain/sem_ir/name_scope.h"
+
+namespace Carbon::Check {
+
+// Deduces the effective access kind from the given lookup and lexical access
+// specifiers.
+auto DeduceClangAccess(clang::AccessSpecifier lookup_access,
```

Comment:
We've generally been saying "Deduce" in the toolchain specifically for generic deduction. Perhaps there's a better term here, because it's just doing a combination of the two parameters. Maybe "Compute" or "Combine"?

Reply:
Done.